synapse/lib/ast.py

```diff
@@ -2864,22 +2889,42 @@ async def walkNodeEdges(self, runt, node, verb=None):
 
     async def run(self, runt, genr):
 
-        const = None
-        if isinstance(self.kids[0], Const):
-            const = self.kids[0].value()
+        @s_cache.memoize(size=100)
+        def isDestForm(formname, destforms):
+
+            if not isinstance(destforms, (list, tuple)):
```

Comment:
type error. We can't have destforms be a list error since a list is unhashable and we cannot memoize it.